src/GenAPI/Microsoft.DotNet.GenAPI/GenAPIApp.cs

```diff
@@ -198,9 +201,17 @@ private static string ReadHeaderFile(string? headerFile)
 
             """;
 
-            return !string.IsNullOrEmpty(headerFile) ?
+            string header = !string.IsNullOrEmpty(headerFile) ?
```

Comment:
This looks like the best minimal change.  Something we might consider in the future is making CSharpFileBuilder responsible for adding the header as syntax, then this could be handled by NormalizeWhitespace.

Reply:
I tried that initially but couldn't get the syntax tree accept arbitary input. The license header file's content might contain single, multi-line comments or something totally different.